js/index.js

```diff
@@ -79,6 +81,15 @@ const displayContent = ({ data }) => {
   //used destructuring for parsed.data => parsed object is destructured and {data} takes it's value.
   let id = 0;
   htmlContainerElement.innerHTML = '';
+  appendLoop(data, id);
+};
+
+const addContent = ({ data }) => {
+  let id = offset;
+  appendLoop(data, id);
```

Comment:
No need in extra complicating and creating extra variables, you should pass `offset` as argument right away

Reply:
Ok, will redress.